.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+### Directions
+_GitHub Issues are reserved for actionable bug reports and feature requests._
+_General questions should be sent to the influxdb@googlegroups.com mailing list._
```

Comment:
Possibly include the link to the web interface? https://groups.google.com/forum/#!forum/influxdb